source/blog/2017-11-15-migrating-kubernetes-on-fedora-atomic-host-27.html.md

```diff
@@ -0,0 +1,104 @@
+
+### System containers for master nodes
+
+```bash
+# atomic install --system --system-package=no --name kube-apiserver registry.fedoraproject.org/f27/kubernetes-apiserver
+
+# atomic install --system --system-package=no --name kube-controller-manager registry.fedoraproject.org/f27/kubernetes-controller-manager
+
+# atomic install --system --system-package=no --name kube-scheduler registry.fedoraproject.org/f27/kubernetes-scheduler
+```
+
+Note: the kube-apiserver system container provides the `kubectl` client.
+
+
+### System containers for worker nodes
+
+```bash
+# atomic install --system --system-package=no --name kubelet registry.fedoraproject.org/f27/kubernetes-kubelet
+
+# atomic install --system --system-package=no --name kube-proxy registry.fedoraproject.org/f27/kubernetes-proxy
+```
+
+### System container for etcd
+
+```bash
+# atomic install --system --system-package=no --storage=ostree --name etcd registry.fedoraproject.org/f27/etcd
+```
+
+When installed with the name "etcd," the etcd system container expects to find stores etcd data in `/var/lib/etcd/etcd.etcd`. The etcd rpm is configured by default to store data in `/var/lib/etcd/default.etcd`, and the ansible scripts in [kubernetes/contrib](https://github.com/kubernetes/contrib/tree/master/ansible) use `/var/lib/etcd`. On a system running etcd as configured by the kubernetes/contrib ansible scripts, you'd move your data as follows:
+
+```bash
+# systemctl stop etcd
+
+# cp -r /var/lib/etcd/member /var/lib/etcd/etcd.etcd/
+```
+
+Note: the etcd container provides the `etcdctl` client.
+
+### System container for flannel
+
+```bash
+# atomic install --system --system-package=no --name flanneld registry.fedoraproject.org/f27/flannel
+```
+
+### Updating system containers
+
+System container updates are independent of host updates. You can update a system container by pulling an updated version of the image, and then running the `atomic containers update` command.
+
+```
+# atomic pull registry.fedoraproject.org/f27/etcd
+# atomic containers update etcd
+```
+
+ You can then roll back to the previous system container version by running `atomic containers rollback`.
+ 
+ ```
+ # atomic containers rollback etcd
+ ```
+
+## Migrating Kubernetes and related components using Package Layering
+
+You won't be able to layer new package versions on until the existing ones are removed during the [upgrade to 27](http://www.projectatomic.io/blog/2017/11/fedora-atomic-26-to-27-upgrade/).  Upon rebooting in to 27, you can run the following commands to replace Kubernetes, Flannel and Etcd with package layering. after which your components and any cluster based on them should be up and running. 
```

Comment:
is it possible to `rpm-ostree rebase remote:ref --install kubernetes-master flannel` ?